homework-3/src/test/java/RandomString.java

```diff
@@ -0,0 +1,31 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class RandomString {
+    public static void main(String[] args) {
+        String[] stringsList = {"Hello World", "Have a great day", "Angel", "Random String", "Word Game"};
+        List<String> filteredStringList = new ArrayList<String>();
+
+
+        for(Integer i = 0; i < stringsList.length; i++) {
+            String item = stringsList[i];
+            Integer itemLength = item.length();
+
+            if (6 <= itemLength && itemLength <= 15) {
+                filteredStringList.add(item);
+            }
+        }
+
+        generateRandomString(filteredStringList);
+    }
+
+    static String generateRandomString(List<String> list) {
+        Random randomList = new Random();
+        Integer filteredStringListItem = randomList.nextInt(list.size());
+        String randomElement = list.get(filteredStringListItem);
+
+        System.out.println(randomElement);
+        return randomElement;
+    }
+}
```

Comment:
that's not what we want to do.
We want a single method that **generates and return a list**.
**parameter should be an int** that represents the number of strings that we want to generate.

method logic:
for **X** times generate a **random** String with random length (from 6 to 15 symbols) and add them to the list that you should return, where **X** - is a parameter.

So in main method there should be only 2 lines:
1. List<String> randomStringsList = generateRandomStringsList(X);
2. print that list